P5/Source/Specs/localProp.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?><!-- Copyright TEI Consortium. Dual-licensed under CC-by and BSD2 licences See the file COPYING.txt for details $Date$ $Id$ -->
+<?xml-model href="http://jenkins.tei-c.org/job/TEIP5-dev/lastSuccessfulBuild/artifact/P5/release/xml/tei/odd/p5.nvdl" type="application/xml" schematypens="http://purl.oclc.org/dsdl/nvdl/ns/structure/1.0"?>
+<elementSpec ident="localProp" module="gaiji" xml:id="LOCALPROP" xmlns="http://www.tei-c.org/ns/1.0" xmlns:rng="http://relaxng.org/ns/structure/1.0"
+    xmlns:sch="http://purl.oclc.org/dsdl/schematron">
+    <gloss versionDate="2018-08-22" xml:lang="en">locally defined property</gloss>
+    <desc versionDate="2018-08-22" xml:lang="en">holds the name and value of a locally defined character (or glyph) property as part of its attributes.</desc>
```

Comment:
```suggestion
    <desc versionDate="2020-01-28" xml:lang="en">provides a locally defined character (or glyph) property.</desc>
```
The fact that these are provided as a name-value pair or via attributes does not really belong in the description.